modules/sdk-coin-zeta/test/unit/zeta.ts

```diff
@@ -268,7 +268,7 @@ describe('Zeta', function () {
             amount: 'UNAVAILABLE',
           },
         ],
-        outputAmount: 'UNAVAILABLE',
```

Comment:
any specific reason why we changed this ?

Reply:
I didn't see any usage of it and made is same as atom